README.md

```diff
@@ -45,161 +37,60 @@ Therefore, please kindly ensure your project includes the following dependencies
 
-```shell
-npm i @emotion/css @emotion/react @emotion/styled @material-ui/core @material-ui/icons @material-ui/lab react react-dom
 ```
+// with npm
+npm i @emotion/css @emotion/react @emotion/styled @material-ui/core @material-ui/icons @material-ui/lab react react-dom
 
-or
-
-```shell
+// with yarn
 yarn add @emotion/css @emotion/react @emotion/styled @material-ui/core @material-ui/icons @material-ui/lab react react-dom
 ```
 
-### Demo
-
-`czifui` comes with [Storybook](https://storybook.js.org/) integration, so you can browse the components locally by following the steps below:
-
-1. Cloning the repo: `git clone git@github.com:chanzuckerberg/sci-components.git`
-1. Run: `yarn && yarn start`
-1. A new browser tab will be automatically opened with the storybook!
-
-![ ](https://user-images.githubusercontent.com/6309723/124010513-c09f0900-d993-11eb-8fc7-f66a0b4ec16e.png)
-
-#### Tips
-
-It's super useful to read the `*.stories.tsx` files to see how the components are used in real examples. You can also find additional examples in [Aspen](https://github.com/chanzuckerberg/aspen), which uses `czifui` extensively
-
-## Default Theme
+## Usage
 
-### Instructions
+`czifui` comes with four main exports that help you build your app:
 
-To use the default theme, please do the following:
+1. Components - Accessible and reusable components
 
-1. Add the following HTML to your `index.html` at the `<head>` section:
-
-   ```html
-   <link rel="preconnect" href="https://fonts.gstatic.com" />
-   <link
-     href="https://fonts.googleapis.com/css2?family=Open+Sans:ital,wght@0,300;0,400;0,600;0,700;0,800;1,400;1,600;1,700&display=swap"
-     rel="stylesheet"
-   />
-   ```
-
-1. Import the default theme object and use it in Material UI's `<ThemeProvider />`:
-
-   ```tsx
-     import { defaultTheme } from "czifui";
-     import { StylesProvider, ThemeProvider } from "@material-ui/core/styles";
-      import { ThemeProvider as EmotionThemeProvider } from "@emotion/react";
-
-    <StylesProvider injectFirst>
-      <EmotionThemeProvider theme={defaultTheme}>
-        <ThemeProvider theme={defaultTheme}>
-          <YourApp />
-        </ThemeProvider>
-      </EmotionThemeProvider>
-    </StylesProvider>
-   ```
-
-1. Optional: If you want to override the default theme, please use `defaultAppTheme`, override the options, and then call `createTheme` to generate
-   the full theme object like below. This is needed because `createTheme` generates
-   extra theme variables based on the themeOptions provided, so if you override `defaultTheme` directly, some auxillary theme variables will be based on `defaultAppTheme` instead of your own custom options
-
-```tsx
-  import { ThemeProvider as EmotionThemeProvider } from "@emotion/react";
-  import { defaultAppTheme, makeThemeOptions } from "czifui";
-  import { StylesProvider, ThemeProvider } from "@material-ui/core/styles";
-  import createTheme from "@material-ui/core/styles/createTheme";
-
-  const customTheme = {
```

Comment:
should we keep the custom theme example? I think it's still very useful for setting up a new project!